### DISCUSSION TITLE
Accidental submission of assignments

### DISCUSSION THREAD

**Discussion by glory20h** [seominjoon/kaist-ai605#157] [Q&A] [answered]
Hello professor,

I just found out that I accidentally submitted the files for Assignment 2 on KLMS for Assignment 3, when I should have submitted on Assignment 2.
I wish to submit the files for Assignment 3 using late days, but I can't since the files for Assignment 2 is already there.

1. Would it be possible find the files in Assignment 3 to grade my assignment2 submission? The time submitted is in KLMS so it would prove that the submission wasn't late.
2. Would it be possible to submit the files for Assignment 3 via email instead?

I am so sorry for the hassle, and promise it won't happen again.

**Comment by seominjoon** [comment, marked as answer]
Please email the TA in charge for this kind of issues (not in discussions). I am closing this.
